### PR TITLE
Allow the Sentinel service account to submit redemption requests on behalf of the lender

### PR DESCRIPTION
**Pull request by ljiatu**
1. Add a flag called `autoRedemptionAfterLockup` to `LPConfig` to indicate whether we'll allow the Sentinel service account to add redemption requests on behalf of lenders.
2. If `true`, disable redemption cancellation.

### CODE REVIEW

File: contracts/liquidity/TrancheVaultStorage.sol [hunk 1]
```diff
@@ -36,6 +37,7 @@ contract TrancheVaultStorage {
     /// Senior or junior tranche index.
     uint8 public trancheIndex;
 
+    HumaConfig public humaConfig;
```
Comment (bin-57blocks) [inline contracts/liquidity/TrancheVaultStorage.sol:40]: Can't do this

File: contracts/common/PoolConfig.sol [hunk 2]
```diff
@@ -197,6 +199,7 @@ contract PoolConfig is Initializable, AccessControlUpgradeable, UUPSUpgradeable
         uint16 fixedSeniorYieldInBps,
         uint16 tranchesRiskAdjustmentInBps,
         uint16 withdrawalLockoutInDays,
+        bool autoRedemptionAfterLockup,
```
Comment (bin-57blocks) [inline contracts/common/PoolConfig.sol:202]: mistake?

File: contracts/liquidity/TrancheVault.sol [hunk 3]
```diff
@@ -313,11 +319,25 @@ contract TrancheVault is
...
+     * @custom:access Only the lender and the Sentinel service account can request redemption.
      */
-    function addRedemptionRequest(uint256 shares) external {
+    function addRedemptionRequest(address lender, uint256 shares) external {
         if (shares == 0) revert Errors.ZeroAmountProvided();
+        LPConfig memory lpConfig = poolConfig.getLPConfig();
+        if (msg.sender != lender) {
+            if (
+                lpConfig.autoRedemptionAfterLockup &&
+                msg.sender != poolConfig_.humaConfig().sentinelServiceAccount()
```
Comment (bin-57blocks) [inline contracts/liquidity/TrancheVault.sol:334]: Why is it poolConfig_?

**Review by bin-57blocks — APPROVED** [review]

File: contracts/liquidity/TrancheVaultStorage.sol (hunk 1 above)
Comment (ljiatu) [reply contracts/liquidity/TrancheVaultStorage.sol]: Removed

File: contracts/liquidity/TrancheVault.sol (hunk 3 above)
Comment (ljiatu) [reply contracts/liquidity/TrancheVault.sol]: It's a typo

**Pull request merged**